widgetssdk/src/main/java/com/glia/widgets/core/engagement/domain/model/ChatHistoryResponse.kt

```diff
@@ -2,5 +2,5 @@ package com.glia.widgets.core.engagement.domain.model
 
 internal data class ChatHistoryResponse(
     val items: List<ChatMessageInternal>,
-    val newMessagesDividerIndex: Int = -1
+    val newMessagesCount: Int = -1
```

Comment:
To me, it looks a bit strange that the count is negative. I would expect to see 0 instead.

Reply:
fixed [d9a95cf](https://github.com/salemove/android-sdk-widgets/pull/578/commits/d9a95cf17ddc59bf167997cda61337a5ead43dd6)